mantaray/views.py

```diff
@@ -155,6 +156,9 @@ def add_notification(self, popup_text: str) -> None:
         self.notification_count += 1
         self._update_view_selector()
         self.irc_widget.event_generate("<<NotificationCountChanged>>")
+        if self.server_view.audio_notification:
+            playsound("mantaray/audio/notify.mp3", False)
```

Comment:
It would be good to wrap this in try/except, just in case playing the sound fails. Even if something unexpected goes wrong, the user should still see the popup and be able to just continue using mantaray.

In fact, this errors on linux, and the error prevents you from seeing the popup:

```
Exception in Tkinter callback
Traceback (most recent call last):
  File "/usr/lib/python3.9/tkinter/__init__.py", line 1892, in __call__
    return self.func(*args)
  File "/usr/lib/python3.9/tkinter/__init__.py", line 814, in callit
    func(*args)
  File "/home/akuli/mantaray/mantaray/views.py", line 448, in handle_events
    channel_view.add_notification(f"<{event.sender}> {event.text}")
  File "/home/akuli/mantaray/mantaray/views.py", line 160, in add_notification
    playsound("mantaray/audio/notify.mp3", False)
  File "/home/akuli/mantaray/env/lib/python3.9/site-packages/playsound.py", line 80, in _playsoundNix
    raise NotImplementedError(
NotImplementedError: block=False cannot be used on this platform yet
```

This is my fault though. I wrote that `raise NotImplementedError(` code.

Reply:
Turns out that this is fixed in playsound 1.3.0. Just change the version in requirements.txt.